test/functional/apps/discover/_source_filters.ts

```diff
@@ -40,6 +40,10 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       // and load a set of makelogs data
       await esArchiver.loadIfNeeded('logstash_functional');
 
+      await kibanaServer.uiSettings.update({
+        'discover:searchFieldsFromSource': true,
```

Comment:
This test doesn't really make sense without reading fields from source. We should discuss if this functionality should exist when retrieving fields from the fields API.